### PR TITLE
Fix wrong link to Contributing Guide

### PR DESCRIPTION
**Pull request by micksatana**
## Motivation

Fix the wrong link

## Test Plan

This is not related to code, not testing required. Only correct the link.

## Related PRs

None

## Related Issue

#340

### CODE REVIEW

**Comment by micksatana** [comment]
Signed.

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,5 @@ website/node_modules
 website/package-lock.json
 website/translated_docs
 website/yarn.lock
+
+.idea/
```
Comment (JoelMarcey) [inline .gitignore]: Hi @micksatana -- what is this? Where is it coming from?

Comment (micksatana) [reply .gitignore]: Hi @JoelMarcey thanks for the review.

I added so anyone using IDE from intellij will not accidentally commit into the code

**Comment by JoelMarcey** [comment]
@micksatana Hi. Sorry, I got a bit sidetracked on other stuff today. I was ok in putting `.idea` in `.gitignore`. But maybe we can put that in a separate PR anyway.

Thanks for this PR.

**Pull request merged**